driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -117,7 +114,24 @@ public String getQueryString() {
 
     @Override
     public ByteBuffer[] getValues() {
```

Comment:
Woops,  I should have realized that.  I noticed some of this functionality was already in 2.1, but I didn't put the pieces completely together.  That makes sense, thanks!